Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Prefabs/Placard.prefab

```diff
@@ -197,9 +199,7 @@ TextMesh:
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1937578906433374}
-  m_Text: 'Fluent
-
-    Unlit'
+  m_Text: 
```

Comment:
Why this text change?

Reply:
Because it's the prefab. The text should be empty as it's set differently for every part in the scene that uses it